install-linux.sh

```diff
@@ -81,6 +85,20 @@ You must specify the VANTA_KEY environment variable in order to install the agen
     exit 1
 fi
 
+if [ ! -z "$VANTA_EXPERIMENTAL_SELINUX" ] && [ -z "$VANTA_NOSTART" ]; then
+    printf "\033[31m
+You must specify the VANTA_NOSTART environment variable when using VANTA_EXPERIMENTAL_SELINUX.
+\n\033[0m\n"
+    exit 1
+fi
+
+if [ "${OS}" == "Redhat" ] && [ ! -z "$VANTA_EXPERIMENTAL_SELINUX" ]; then
+    printf "\033[31m
+SELinux support is not available on your OS.
+\n\033[0m\n"
+    exit 1
+fi
+
```

Comment:
Can we consolidate this, plus lines 118-127 into one block all conditioned by VANTA_EXPERIMENTAL_SELINUX being specified?

i.e. 

if experimental selinux:
    if not vanta_nostart: error
    if not redhat: error
    curl the selinux package
endif

Just a bit easier to reason about, and replace when it's no longer experimental.

Probably not worth doing the actual install in the same block, as that adds dependencies with the rest of the script. But the 3 blocks here can be consolidated into 1.

Reply:
@ostrowr I added screenshots to the PR description.

I tested the original script on a clean centos box (substituting the download URL with the staging one and commenting our the checksum test), and I just retested these changes locally on ubuntu.